scenarios/perf-eval/cluster-automatic/terraform-inputs/azure.tfvars

```
@@ -0,0 +1,22 @@
+scenario_type  = "perf-eval"
+scenario_name  = "api_cli_test"
+deletion_delay = "72h"
+owner          = "aks"
+aks_cli_config_list = [
+  {
+    role                          = "client1"
+    aks_name                      = "test"
```

```suggestion
    role                          = "client"
    aks_name                      = "aks-automatic"
```
Can you use this names instead of test, so that when we see a cluster we know it's from this test scenario.